server/email_verification.go

```diff
@@ -236,6 +328,12 @@ func handleEmailVerifyFunc(verifiedTpl *template.Template, issuer url.URL, keysF
 			}
 			return
 		}
+
+		qr := cbURL.Query()
+		qr.Set("code", code)
+		qr.Set("state", ses.ClientState)
```

Comment:
ok, now I get it, thanks. we have a problem here :P
what would you do in this piece of code then?